Modules/IO/NIFTI/src/itkNiftiImageIO.cxx

```diff
@@ -521,6 +524,36 @@ CastCopy(float * to, const void * from, size_t pixelcount)
   }
 }
 
+// Internal function to convert vectors between RAS and LPS coordinate systems.
+// Dimensions are CXYZT (ITK memory layout)
+template <typename TBuffer>
+void
+ConvertRASToFromLPS_CXYZT(TBuffer * buffer, size_t size)
+{
+  size_t numberOfVectors = size / 3;
+  for (size_t i = 0; i < numberOfVectors; ++i)
+  {
+    buffer[0] *= -1;
+    buffer[1] *= -1;
+    buffer += 3;
+  }
+}
+
+// Internal function to convert vectors between RAS and LPS coordinate systems.
+// Dimensions are XYZTC (NIFTI memory layout)
+template <typename TBuffer>
+void
+ConvertRASToFromLPS_XYZTC(TBuffer * buffer, size_t size)
```

Comment:
@lassoan Can you please place both `ConvertRASToFromLPS_CXYZT` and `ConvertRASToFromLPS_XYZTC` inside an unnamed namespace? (Possibly nicely together in the same namespace.) That's just where any local helper function that is defined in a cxx file is supposed to be. Sorry it's not my fault, blame C++  😺 

Just a matter of adding right before their function definition:

    namespace
    {

and right afterwards:

    } // namespace

So the code itself does not need to move.

Reply:
I was surprised why all the existing helper functions there were not in an unnamed namespace, but I wanted to keep changes to existing code to the minimum. I've added the namespace now.